components/brave_wallet/common/features.cc

```diff
@@ -23,6 +23,8 @@ const base::Feature kBraveWalletFilecoinFeature{
 
 const base::Feature kBraveWalletSolanaFeature{"BraveWalletSolana",
                                               base::FEATURE_ENABLED_BY_DEFAULT};
+const base::FeatureParam<bool> kCreateDefaultSolanaAccount{
+    &kBraveWalletSolanaFeature, "create-default-solana-account", true};
```

Comment:
What should the naming convention be here? `CreateDefaultSolanaAccount` or `create_default_solana_account` or `create-default-solana-account`?

Reply:
Duno, i've seen CamelCase and both "-" and "_" separated param names both in brave and chromium code, seems there is no naming conventions here. Changed to "_" cause it happens more often.